src/main/java/com/oltpbenchmark/api/BenchmarkModule.java

```diff
@@ -91,6 +104,20 @@ public final Connection makeConnection() throws SQLException {
         }
     }
 
+    private HikariDataSource hikariDataSource;
+
+    public void createDataSource() {
+        HikariConfig config = new HikariConfig();
+        config.setJdbcUrl(workConf.getUrl());
+        if (!StringUtils.isEmpty(workConf.getUsername())) {
+            config.setUsername(workConf.getUsername());
+            config.setPassword(workConf.getPassword());
+            config.setMaximumPoolSize(workConf.getTerminals());
```

Comment:
I think we should have a separate property for max pool size. It should be separate from the number of terminals.

Reply:
added ```max_pool_size``` which will be taken from the input yaml/xml. If not provided, default will be set to terminals.